tools/myst/myst.edl

```diff
@@ -123,7 +135,13 @@ enclave
             uint64_t self_event,
             [in] const struct myst_timespec* timeout);
 
-        long myst_sched_yield_ocall();
+        long myst_sched_yield_ocall()
+            transition_using_threads;
+
+        long myst_sched_getparam_ocall(
+            pid_t pid,
+            [out] struct myst_sched_param* param)
```

Comment:
why can this not be just `struct sched_param`? You can include the correct header in the top of the file I believe?

Reply:
I tried adding #include "sched.h", "myst/sched.h" and "sys/sched.h" and none of them worked as expected. Do you know the right header?

Reply:
man page suggests just `#include <sched.h>`

Reply:
Tried, didn't work either..